section1/reflection.md

```diff
@@ -1,19 +1,32 @@
 
-3. How would you log the string `"Hello World!"` to the console?
+3. **How would you log the string `"Hello World!"` to the console?**
+`console.log("Hello World!");`
 
-4. What is/are the character(s) you would use to indicate comments in a JavaScript file? What is the purpose of a code comment?
+4. **What is/are the character(s) you would use to indicate comments in a JavaScript file? What is the purpose of a code comment?**
+// for single line comments and /* text */ for multi-line comments. The purpose of a code comment is include details and explanations that we don't want logged to the console. It could also be for broken or unused code.
 
-5. In your own words, what is a variable? How would you explain it to a 5 year old?
+5. **In your own words, what is a variable? How would you explain it to a 5 year old?**
+A variable is used when we need to change the data of a value. It allows us to change this data flexibly and often. If I were to explain it to a 5 year old, I might compare it to a glass holding a drink. The glass is like the variable, and the drink is the value. The drink can be changed at any time to a different drink.
 
-6. Think of a site or app you use frequently. What are three variables that are probably used? Which data type would each of those variables probably hold?
+6. **Think of a site or app you use frequently. What are three variables that are probably used? Which data type would each of those variables probably hold?**
+On Reddit, string is used for the name of a subreddit. Number is used for the amount of upvotes on a post. Boolean is used when asked if you want to exit without saving or uploading a post you are in the middle of composing.
 
-7. In your own words, explain what concatenation is.
+7. **In your own words, explain what concatenation is.**
+Concatenation allows us to log a string containing string and variables to the console. Within the string, you use a `+` to combine smaller string and variables with assigned values, which will then form a complete sentence.  
 
-8. Think of a site or app you use frequently. Where do you think the developers used concatenation?
+8. **Think of a site or app you use frequently. Where do you think the developers used concatenation?**
+On the Chase banking app, the developers use concatenation in statements updating the balance in your checking account i.e.
+`var amount = 200`
+`console.log("You have $" + amount + " in your checking account.")`
 
-9. What questions do you still have about the work we've done so far? (not having a question is not an option)
+9. **What questions do you still have about the work we've done so far? (not having a question is not an option)**
+I am still curious about when would be a better time to use concatenation vs. interpolation. I am also still having a little trouble wrapping my head fully around booleans and how to utilize them effectively when defining variables and logging to the console.
```

Comment:
Concat vs Interp - purely preference. Most devs use interpolation as it is newer and most find it's easier to use.

Booleans - I'm going to say, give them some time and through building projects that require them in Mod 1, you'll be really clear on when you need them!